SabberStoneCore/src/CardSets/Standard/BoomsdayCardsGen.cs

```diff
@@ -862,12 +860,24 @@ private static void MageNonCollect(IDictionary<string, Power> cards)
 			// - TAG_ONE_TURN_EFFECT = 1
 			// --------------------------------------------------------
 			cards.Add("BOT_531e", new Power {
-				Enchant = new Enchant(new Effect(GameTag.SPELLPOWER, EffectOperator.ADD, 2)),
-				Trigger = new Trigger(TriggerType.AFTER_PLAY_CARD)
+				// Seems like there are some problems with Enchant SPELLPOWER effects not being removed
+				Aura = new Aura(AuraType.CONTROLLER, new Effect(GameTag.SPELLPOWER, EffectOperator.ADD, 2))
```

Comment:
This one caused me a lot of problems, probably related to my previous spell changes caused some issue.  If using an Enchant instead of Aura, the the remove trigger subtracts 2 spell damage from the source? ie fireball.  Instead of subtracting from the controller.  Switching to Aura works around this.